superset/__init__.py

```diff
@@ -230,7 +230,10 @@ def is_feature_enabled(feature):
 if conf.get('ENABLE_FLASK_COMPRESS'):
     Compress(app)
 
-Talisman(app, content_security_policy=None)
+if app.config['TALISMAN_ENABLED']:
+    talisman_config = app.config.get('TALISMAN_CONFIG') or \
+                      {'content_security_policy': None}
```

Comment:
Any reason to have `{'content_security_policy': None}` as a fallback? Wouldn't one simply specify the options they require in their `superset_config.py` file?